deps.edn

```diff
@@ -86,10 +86,11 @@
               "-Dclojure.spec.check-asserts=true"]}
 
   :test
-  {:extra-paths ["dev" "test"]
```

Comment:
This "dev" dependency was creating a strange dependency error when reloading namespaces withing test-refresh. I think it was there in testing mainly because it contains a logback.xml file, which is still good for anything REPL related.